community/php7/APKBUILD

```diff
@@ -536,7 +544,7 @@ pear() {
 
 common() {
 	pkgdesc="$pkgdesc (common config)"
-	provides="$pkgname-zlib"  # for backward compatibility
+	provides="$pkgname-zlib php-zlib"  # for backward compatibility
```

Comment:
same here - "php7-common" & "php5-common" packages should have `php-common`, is not it?